adapters/adnuntius/adnuntius.go

```diff
@@ -39,6 +39,14 @@ type Ad struct {
 		Amount   float64
 		Currency string
 	}
+	NetBid struct {
+		Amount   float64
+		Currency string
+	}
+	GrossBid struct {
+		Amount   float64
+		Currency string
```

Comment:
@mikael-lundin  could you help to understand how values are set to `NetBid.Currency` and `GrossBid.Currency`. If these vars are not in use then can should remove them

Reply:
They are always returned from the ad-server so they will always be included in the response.

Reply:
But yeah you are right they can be removed. :)